ArmVirtPkg/ArmVirtKvmTool.fdf

```diff
@@ -173,8 +173,7 @@ READ_LOCK_STATUS   = TRUE
   INF OvmfPkg/VirtioScsiDxe/VirtioScsi.inf
   INF OvmfPkg/VirtioRngDxe/VirtioRng.inf
 
-  INF ShellPkg/Application/Shell/Shell.inf
-  INF ShellPkg/DynamicCommand/TftpDynamicCommand/TftpDynamicCommand.inf

```

Comment:
@kraxel - This is probably the most 'invasive' of the changes similar to this, as it's less clear (to me) whether it was intentional that this FDF includes neither of the 'extra' two commands (`VariablePolicyDynamicCommand` and `LinuxInitrdDynamicShellCommand`; as well as not including `HttpDynamicCommand`)

It is hopefully (at worst) harmless (i.e. possibly it's a good thing, after all!) if these are included. If not then an additional define to disable these (possibly only actually used in this specific platform) could be added.

Reply:
On further examination as to what ArmVirtKvmTool is (i.e. just a version of ArmVirtPkg to run in the lightweight kvmtool hypervisor) then it seems fairly unlikely that there is any issue synchronising the commands here with the other OvmfPkg and ArmVirtPkg platforms.

Reply:
PS Of course it is tested to build and run fine in CI. If I grab a chance I'll check it by hand, but it seems fine.